.github/workflows/test-only.yml

```diff
@@ -114,7 +114,13 @@ jobs:
             exit 1
           fi
           
-          # @todo - scan the container log to see the right "graceful shutdown" text exists 
+          docker logs -f sig-test 2> /dev/null | grep 'graceful shutdown'
+          # both GNU or BSD grep returns 0 if the line was selected
```

Comment:
EXIT STATUS section of https://man.freebsd.org/cgi/man.cgi?grep
https://www.gnu.org/software/grep/manual/html_node/Exit-Status.html